lblod/frontend-reglementaire-bijlage
# Drop unused ember-data model definitions

## Overview
This PR drops ember-data models which are not used by this application.
Reflection of the equivalent [PR](https://github.com/lblod/app-reglementaire-bijlage/pull/76) on the backend.

##### connected issues and PRs:
https://github.com/lblod/app-reglementaire-bijlage/pull/76

### How to test/reproduce
- Test in combination with https://github.com/lblod/app-reglementaire-bijlage/pull/76
- No functionality of the app should change

### Checks PR readiness
